From 1d30b5cd9156b1a9d409c8810dab26e2dc5065af Mon Sep 17 00:00:00 2001
From: Tony Zorman <soliditsallgood@mailbox.org>
Date: Thu, 29 Dec 2022 10:35:58 +0100
Subject: [PATCH] Bump bytestring dependency

This fixes the build for GHC 9.2, which ships with a higher version of
the bytestring library.
---
 debug-me.cabal | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debug-me.cabal b/debug-me.cabal
index d5a44e9..2dd60a0 100644
--- a/debug-me.cabal
+++ b/debug-me.cabal
@@ -55,7 +55,7 @@ Executable debug-me
   Build-Depends:
       base (>= 4.9 && < 5.0)
     , network (>= 2.6)
-    , bytestring == 0.10.*
+    , bytestring < 0.12
     , cryptonite (>= 0.20)
     , unix (>= 2.7)
     , process (>= 1.4)
--
2.39.0
